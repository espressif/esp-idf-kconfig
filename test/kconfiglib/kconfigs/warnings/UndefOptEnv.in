mainmenu "Undefined opt=env"

    config IDF_ENV_FPGA
        bool
        option env="IDF_ENV_FPGA_DEFINITELY_NOT_SET" # Should not be set
