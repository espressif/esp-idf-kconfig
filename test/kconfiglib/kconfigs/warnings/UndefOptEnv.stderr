warning: IDF_ENV_FPGA has 'option env="IDF_ENV_FPGA_DEFINITELY_NOT_SET"', but the environment variable IDF_ENV_FPGA_DEFINITELY_NOT_SET is not set
